### PR TITLE
CCD-44871 questions-portal ember localization

### PR DESCRIPTION
**Pull request by rawburt**
This creates an `EmberWriter` for `questions-portal`. 

I also took a first pass at creating specs for the actual running of the writers. It doesn't do much yet but the test will fail if you have broken code, so that's a plus 😄

### CODE REVIEW

File: spec/lib/localio/writers/ember_writer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      expect(described_class.javascript_parsing(string)).to eq "{1} characters remaining of {2}"
+    end
+
+    it "escapes triple quotes" do
+      string = 'No search results in <s$1> for ""<s$2>""'
+      expect(described_class.javascript_parsing(string)).to eq 'No search results in {1} for \"{2}\"'
+    end
+  end
+
+  describe ".write" do
```
Comment (robinheinze) [inline spec/lib/localio/writers/ember_writer_spec.rb:21]: yay!

**Review by robinheinze — APPROVED** [review]
Look good.

**Comment by amymichellejohnson** [comment]
Yep, looks good!

**Review by amymsahli — APPROVED** [review]
Looks good

**Pull request merged**